apps/ima/views/permission.html

```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
```

Comment:
Do you need a html template for this page? Wouldn't it be created automatically as it is the file name same as the step name?